vscode/src/test/unit/jdkDownloader.unit.test.ts

```diff
@@ -0,0 +1,11 @@
+import { suite, test } from 'mocha';
```

Comment:
can we pick some small file in source code and add one meaningful test ? would that work